nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -301,8 +328,7 @@ def _apply(self,
             node = nncf_graph.get_node_by_name(target_node_name)
             if quantization_target_point.type == TargetType.OPERATION_WITH_WEIGHTS:
                 try:
-                    input_tensor_names, _ = self._backend_entity.get_tensor_names(node)
-                    weight_initializer_name = input_tensor_names[1]
+                    weight_initializer_name = self._backend_entity.get_weight_tensor_name(model, node.node_name)
                     weight_tensor = self._backend_entity.get_initializer_value(model, weight_initializer_name)
```

Comment:
To make the logic common to all backends, it would be better to use `port_id` instead of `weight_initializer_name`.

Reply:
We need edge name to filter some quantizers which put in the shared weight. In that case they have common edge name but the different node names. The only way to do filtration is to keep edge names. 
Do you have other idea?

Reply:
Yes, it should be. Good suggestion!